hilti/toolchain/include/ast/attribute.h

```diff
@@ -96,15 +163,59 @@ class Attribute : public Node {
     }
 
 protected:
-    Attribute(ASTContext* ctx, Nodes children, std::string tag, Meta m = Meta())
-        : Node(ctx, NodeTags, std::move(children), std::move(m)), _tag(std::move(tag)) {}
+    Attribute(ASTContext* ctx, Nodes children, std::string tag, Kind kind, Meta m = Meta())
+        : Node(ctx, NodeTags, std::move(children), std::move(m)), _tag(std::move(tag)), _kind(kind) {}
 
     std::string _dump() const override;
 
     HILTI_NODE_0(Attribute, final);
 
 private:
     std::string _tag;
```

Comment:
This should be redundant for any successfully parsing `Attribute` since we can recompute it from `_kind`.